PendingReleaseNotes.md

```diff
@@ -21,6 +21,9 @@ an example usage
 - `NodeAffinity` can be applied to `rook-ceph-agent DaemonSet` with `AGENT_NODE_AFFINITY` environment variable.
 - `NodeAffinity` can be applied to `rook-discover DaemonSet` with `DISCOVER_AGENT_NODE_AFFINITY` environment variable.
 - Rook does not create an initial CRUSH map anymore and let Ceph do it normally
+- Flexvolume plugin now support dynamic PVC resizing.
+
+### <Storage Provider>
```

Comment:
is this storage provider section needed?

Reply:
I think this is here by accident. I will remove it.